.github/ISSUE_TEMPLATE/13_test_failure.yml

```diff
@@ -0,0 +1,57 @@
+name: "[TEST] Test failure"
+labels: ["test-failure"]
+description: "[TEST] (Internal) For reporting test failures on the nightly builds"
+title: "Failing test(s): TestAccWhatever"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This is a template for reporting test failures on nightly builds. It should only be used by core contributors who have access to our CI/CD results.
+  - type: textarea
+    id: impacted-tests
+    attributes:
+      label: Impacted tests
+      description: List all impacted tests for searchability. The title of the issue can instead list one or more groups of tests, or describe the overall root cause.
+      value: |
+        - TestAccWhatever
+    validations:
+      required: true
+  - type: textarea
+    id: affected-resources
+    attributes:
+      label: Affected Resource(s)
+      description: List the primary resource or data source under test, even if the root cause of the failure is in a different resource. Use `google_*` for tests related to core provider logic.
+      value: |
+        - google_XXXXX
+    validations:
+      required: true
+  - type: textarea
+    id: failure-rates
+    attributes:
+      label: Failure rates
+      description: For 100% test failures, provide the start date. For flakey tests, use as much of the test history as seems relevant.
+      value: |
+        - 100% since YYYY-MM-DD
+        - X% as of YYYY-MM-DD (Y/120 runs)
```

Comment:
I'm not sure how best to convey this but basically, I recently noticed that TeamCity summarizes the entire history of test runs (which is usually ~120) on the test history page, which means that you don't have to manually count the numbers in a particular month (unless the failure started recently): https://hashicorp.teamcity.com/test/-5152197187456236967?currentProjectId=TerraformProviders_GoogleCloud_GOOGLE_BETA_NIGHTLYTESTS&expandTestHistoryChartSection=true

![Screenshot 2024-02-27 at 9 36 05 AM](https://github.com/hashicorp/terraform-provider-google/assets/299979/6e07ac1b-24b1-40ef-aad9-d97adaef0e20)

Possibly this is excessive / too prescriptive though?

Reply:
Oh I see, yea that is a lot easier than getting the % for a month. Unfortunate that it is a success rate instead of failure rate though, so we have to do math :/. Maybe we could just be more explicit, like `X failed of last Y runs`?

I also misread the date as being "since", and I do think it is important to capture some sense of how many runs we are looking at. So maybe we do need all of this info ("as of" date, and number of runs).

In any case, I understand the dilemma now. Happy to move forward with what you have if you don't see an adjustment that makes it better.

Reply:
tentatively going with `X% (Y failed out of last Z runs) as of YYYY-MM-DD` for now... we can always tweak it further later.